### DISCUSSION TITLE
Rtu Turbine #2 is not showing up on monitor

### DISCUSSION THREAD

**Discussion by Yuki208** [MikaylaFischler/cc-mek-scada#529] [Support] [answered]
I already have one turbine connected to the rtu and it works but when I connected the second one to my rtu it will not show up on the monitor
![2024-07-26_15 25 00](https://github.com/user-attachments/assets/e1dd78c7-c840-4ed4-8883-4f7077bf4988)
![2024-07-26_15 24 54](https://github.com/user-attachments/assets/7ff77e87-e2f8-4482-b2c0-2c96de4fff9a)

**Comment by MikaylaFischler** [comment, marked as answer]
You need to update your cooling configuration on the supervisor, otherwise it doesn't expect to see other turbines

**Reply by Yuki208** [reply]
That was the issue thanks for the help.
